### PR TITLE
Update dependencies and import command

### LINKED ISSUE #183: rails r 'Importer.run' exits immediately, nothing is scraped
Hey when I try running `rails r 'Importer.run'` nothing gets logged to **logs/development.log** and the db is empty.

### PR DESCRIPTION
**Pull request by jedahan**
I have only tested this on my local macOS 10.13.4 install

Fixes #183 and #181

### CODE REVIEW

**Comment by edap** [comment]
Hello @jedahan , is there any reason why you bumped a new ruby version (as stated in the Gemfile)?
I had the same issue, the ruby version in the readme was not the same as that one in the Gemfile, I have updated the readme and added a .ruby-version file. We should try to make the enviroment consistent and documented, so that is easier to jump on the development.

**Comment by jedahan** [comment]
I changed it to >= because my Mac didn't have 2.2.4 and it wouldn't install. It works on 2.5.1 but I didn't want to specify that number exactly since I assume it still works on 2.2.4

**Comment by edap** [comment]
Ok, but if your mac does not have the ruby version required by the Gemfile, please install it. If every developer start to bump a new ruby version depending on its local environment, we end up in a big mess, because it is really easy to brake gem dependencies. Let me know if you are in trouble
To manage different ruby version, use rbenv, it is straightforward https://github.com/rbenv/rbenv

**Comment by jedahan** [comment]
It would also be nice if we could support whatever version of ruby is installed (that this has been tested on), so that we don't have to ask every developer to install a new manager just for ruby. If it works on 2.5.1, why shouldn't we allow developers to use that version?

**Comment by edap** [comment]
"It would also be nice if we could support whatever version of ruby is installed (that this has been tested on), so that we don't have to ask every developer to install a new manager just for ruby."

I'm not in favour of this solution, I have worked with ruby for ~8 years and I can tell you that it is better to have one supported version(that one that is used in production) with its gemfile than 10 ruby version, because getting the developers all agree with one ruby version is much more easier than getting a website working with differens ruby version.

rbenv is one line away from your command line, `brew install rbenv`. And it is the best way to install and keep ruby up to date.

**Comment by jedahan** [comment]
It was definitely my mistake putting 2.5.1 in the lock.

If the code works on 2.5.1 and 2.2.4, what is bad about with committing 2.2.4 on the Gemfile.lock and having the Gemfile specify 2.2.4 through 2.5.1? 

This keeps production stable, and makes it easier for contributors (especially first time) to just start working with the code. Experienced developers can use whatever version manager they want for more stability, without adding friction for those who don't have rbenv/rvm/chruby installed and setup, or even homebrew installed.

We definitely can recommend contributors use a version manager but if its not necessary, imo the less we ask of contributors, the better.

Either way, it looks like this might be rewritten in javascript anyway, so i'm gonna hold off to hear what @atduskgreg 's thoughts are on this.

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: bundle exec rails server -p 5000
```
Comment (jamezilla) [inline Procfile]: this is a defunct way of running things. left over from when we used to run on heroku.

**Pull request closed without merging**